eosiojavaabieos/build.gradle

```diff
@@ -1,145 +1,55 @@
-                    url = 'https://github.com/EOSIO/eosio-java-android-abieos-serialization-provider/'
-                    licenses {
-                        license {
-                            name = 'MIT License'
-                            url = 'https://github.com/EOSIO/eosio-java-android-abieos-serialization-provider/blob/master/LICENSE'
-                        }
-                    }
-                    developers {
-                        developer {
-                            id = 'block.one'
-                            name = 'Block One'
-                            email = 'dev@block.one'
-                        }
-                    }
-                }
-
-                pom.withXml {
-                    final dependenciesNode = asNode().appendNode('dependencies')
+publishing {
+    publications {
+        jar(MavenPublication) {
+            from components.java
 
-                    ext.addDependency = { dep, scope ->
-                        if (dep.group == null || dep.version == null || dep.name == null || dep.name == "unspecified")
-                            return //  ignore invalid dependencies
+            artifact sourcesJar {
+                classifier "sources"
+            }
 
-                        final dependencyNode = dependenciesNode.appendNode('dependency')
-                        dependencyNode.appendNode('groupId', dep.group)
-                        dependencyNode.appendNode('artifactId', dep.name)
-                        dependencyNode.appendNode('version', dep.version)
-                        dependencyNode.appendNode('scope', scope)
+            artifact javadocJar {
+                classifier "javadoc"
+            }
 
-                        if (!dep.transitive) {
-                            // If this dependency is transitive, we should force exclude all its dependencies from the POM
-                            final exclusionNode = dependencyNode.appendNode('exclusions').appendNode('exclusion')
-                            exclusionNode.appendNode('groupId', '*')
-                            exclusionNode.appendNode('artifactId', '*')
-                        } else if (!dep.properties.excludeRules.empty) {
-                            // ...otherwise add specified exclude rules
-                            final exclusionNode = dependencyNode.appendNode('exclusions').appendNode('exclusion')
-                            dep.properties.excludeRules.each { rule ->
-                                exclusionNode.appendNode('groupId', rule.group ?: '*')
-                                exclusionNode.appendNode('artifactId', rule.module ?: '*')
-                            }
-                        }
-                    }
+            groupId libraryGroupId
+            version libraryVersion
+            artifactId libraryArtifactId
```

Comment:
The value for libraryArtifactId needs to change in the build.gradle from "**def libraryArtifactId = 'eosiojavaandroidabieosserializationprovider'**" to "**def libraryArtifactId = 'eosiojavaabieos'**".  It's causing the build deploy error you are seeing.